ACE/bin/make_release.py

```diff
@@ -637,7 +637,8 @@ def move_packages (name, stage_dir, package_dir):
     print ("\tZip file...")
     target_file = join (package_dir, name + ".zip")
     shutil.copy (join (stage_dir, "zip-archive.zip"), target_file)
-    ex ("md5sum " + target_file + " > " + target_file + ".md5")
+    ex ("cd " +  package_dir + "; md5sum " + os.path.basename(target_file) + " > " + os.path.basename(target_file) + ".md5")
```

Comment:
Why not use `name`, target_file is build up of a directory and filename, when you put the filename+extension in a new variable you can use that